### ISSUE TITLE
Use PyExifTool for titles.

### ISSUE THREAD

**Issue by btimby** [btimby/fulltext#61]
https://smarnach.github.io/pyexiftool/

PyExifTool uses exiftool in batch mode rather than invoking exiftool for every file.

**Comment by giampaolo** [comment]
How can this be "batched"? Currently `fulltext` deals with a single path (or file obj). Also `duster` approach is "per-file".

**Comment by btimby** [comment]
We won't "batch" files, still process one-at-a-time. That would not change, the difference is:

```python
exiftool /path/to/file1
exiftool /path/to/file2
exiftool /path/to/file3
```
vs.
```python
exiftool -batch
cat /path/to/file1 > /tmp/exiftool.sock
cat /path/to/file2 > /tmp/exiftool.sock
cat /path/to/file3 > /tmp/exiftool.sock
```

Just means that the overhead of creating the `exiftool` process is eliminated (the process is recycled). Still a win even if we use batches of 1. Think of this the way that `unoconv` works, unoconv is a client that connects to LibreOffice/OpenOffice process running as a server. That server process can be persistent, and do many conversions over it's lifetime (instead of just one).

**Comment by giampaolo** [comment]
Implemented by #69. Closing.
